src/ImageSharp/Metadata/Profiles/Exif/Tags/ExifTag.String.cs

```diff
@@ -94,7 +94,7 @@ public abstract partial class ExifTag
         /// <summary>
         /// Gets the MDFileUnits exif tag.
         /// </summary>
-        public static ExifTag<string> MDFileUnits => new ExifTag<string>(ExifTagValue.MDFileUnits);
+        public static ExifTag<string> MDFileUnits { get; } = new ExifTag<string>(ExifTagValue.MDFileUnits);
```

Comment:
Pushed this by accident but it was something I found the other day when looking at the EXIF tags.